packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -3,48 +3,107 @@ import usePersistentContext from '../../hooks/usePersistentContext';
 import { Button } from '../buttons/Button';
 import NotificationsContext from '../../contexts/NotificationsContext';
 import { cloudinary } from '../../lib/image';
+import CloseButton from '../CloseButton';
+
+export enum NotificationPromptSource {
+  NotificationList = 'notificationList',
+  NewComment = 'newComment',
+  NewSource = 'newSource',
+}
 
 const DISMISS_BROWSER_PERMISSION = 'dismissBrowserPermission';
 
-function EnableNotification(): ReactElement {
-  const { hasPermission, requestPermission } = useContext(NotificationsContext);
-  const [dismissed, setDismissed, isLoaded] = usePersistentContext(
-    DISMISS_BROWSER_PERMISSION,
-    false,
-  );
+type DismissBrowserPermissions = {
+  [NotificationPromptSource.NotificationList]: boolean;
+  [NotificationPromptSource.NewComment]: boolean;
+  [NotificationPromptSource.NewSource]: boolean;
+};
 
+const DEFAULT_CACHE_VALUE: DismissBrowserPermissions = {
+  newComment: false,
+  newSource: false,
+  notificationList: false,
+};
+
+type EnableNotificationProps = {
+  source?: NotificationPromptSource;
+  parentCommentAuthorName?: string;
+};
+
+function EnableNotification({
+  source = NotificationPromptSource.NotificationList,
+  parentCommentAuthorName,
+}: EnableNotificationProps): ReactElement {
+  const { hasPermission, requestPermission } = useContext(NotificationsContext);
+  const [dismissedCache, setDismissedCache, isLoaded] =
+    usePersistentContext<DismissBrowserPermissions>(
+      DISMISS_BROWSER_PERMISSION,
+      DEFAULT_CACHE_VALUE,
+    );
+  const dismissed = !!dismissedCache?.[source];
+  const setDismissed = (value: boolean) =>
+    setDismissedCache({ ...dismissedCache, [source]: value });
   if (!isLoaded || dismissed || hasPermission) {
     return null;
   }
-
+  if (source === NotificationPromptSource.NotificationList) {
+    return (
+      <div className="relative py-4 px-6 w-full bg-theme-float border-l typo-callout border-theme-color-cabbage">
+        <span className="mb-2 font-bold">Push notifications</span>
+        <p className="w-3/5 text-theme-label-tertiary">
+          Stay in the loop whenever you get a mention, reply and other important
+          updates.
+        </p>
+        <span className="flex flex-row gap-4 mt-4">
+          <Button
+            buttonSize="small"
+            className="w-28 text-white btn-primary-cabbage"
+            onClick={requestPermission}
+          >
+            Enable
+          </Button>
+          <Button
+            buttonSize="small"
+            className="w-28 btn-tertiary"
+            onClick={() => setDismissed(true)}
+          >
+            Dismiss
+          </Button>
+        </span>
+        <img
+          className="absolute top-2 right-0"
+          src={cloudinary.notifications.browser}
```

Comment:
Did this not get updated with the GIF?

Reply:
I will add it in my next PR (new source)